packages/lint/src/linter.ts

```diff
@@ -36,13 +38,18 @@ function getLinterForLibrary(linter: Linter, library: CadlLibrary<any, any>): Li
   };
 
   function registerRule(rule: LintRule, options?: RegisterRuleOptions) {
-    linter.registerRule(
-      {
-        ...rule,
-        name: `${library.name}/${rule.name}`,
-      },
-      options
-    );
+    const name = `${library.name}/${rule.name}`;
+    linter.registerRule({
+      ...rule,
+      name,
+    });
+    if (options?.autoEnable) {
+      autoEnableRules.push(name);
+    }
+  }
+
+  function autoEnableMyRules() {
```

Comment:
`My` normally doesn't show up in our API names, could we just use `autoEnableRules` or perhaps `enableAllRules`?